test/integration/consul-container/test/envoy_extensions/wasm_test.go

```diff
@@ -0,0 +1,519 @@
+package envoy_extensions
+
+import (
+	"context"
+	sha2562 "crypto/sha256"
```

Comment:
I'll bet that was an autocomplete since I have a function of the same name. I'll update the function name to be more descriptive and change this back.